update: Add steps for custom container API spec (#4170)
aws-amplify/docs: src/pages/cli/usage/containers.mdx

password = os.environ['DB_PASSWORD']
```

> ## Bring Your Own API Specification

Reviewer: ```suggestion
## Bring Your Own Container Specification For Your REST API
```

> When creating an API using containers, you have the option to bring in your own `Dockerfile`(s) and `docker-compose.yml`. When selecting the image you want to use for your REST API, choose `Custom`. Afterward, you can choose when you want to deploy changes to your container.

Reviewer: ```suggestion
When creating an API using containers, you have the option to bring in your own `Dockerfile`(s) and `docker-compose.yml`. When selecting the docker image for your REST API, choose `Custom` and then select when to deploy code and configuration changes to your container.
```

```console
? What image would you like to use Custom (bring your own Dockerfile or docker-compose.yml)
? When do you want to build & deploy the Fargate task 
> On every "amplify push" (Fully managed container source) 
  On every Github commit (Independently managed container source) 
  Advanced: Self-managed (Learn more: docs.amplify.aws/cli/usage/containers) 
```

> The CLI will provide you with some next steps to follow, and you can take your existing specification and allow the CLI to bring it into your project by populating the `src` directory with your specification.

Reviewer: ```suggestion
The CLI will provide you with the next steps to bring your existing docker specification and source files into your project by populating the `src` directory.
```
